### DISCUSSION TITLE
debugger agent is not enabled

### DISCUSSION THREAD

**Discussion by 563256676** [cefsharp/CefSharp#3873] [General]
When I use the RemoveBreakpointAsync method, an error is reported, DevTools Client Error: Debugger agent is not enabled, how should I set it to run normally.

**Comment by amaitland** [comment]
In future please use discussions to ask questions. https://github.com/cefsharp/CefSharp/discussions

Did you call [EnableAsync](http://cefsharp.github.io/api/94.4.x/html/M_CefSharp_DevTools_Debugger_DebuggerClient_EnableAsync.htm) before RemoveBreakpointAsync?
